### DISCUSSION TITLE
Error only occurs with certain parameters of fit

### DISCUSSION THREAD

**Discussion by 265091z** [cvnlab/GLMsingle#115] [Q&A] [answered]
When I ran it on sample data it worked fine, but when I used my fMRI data I got an error.

If all the parameters are set to 1 during fit, an error occurs with cross-validating different numbers of regressors.

If opt['wantlibrary'] is set to 0, no error occurs. Also, if opt['wantlibrary'] is set to 1 and the other parameters are set to 0, no error occurs.

**Is there a problem with my data or design matrix that is causing this error?**

**Comment by kendrickkay** [comment]
We believe this is a code bug with the Python implementation, and not necessarily a problem with your data or design matrix (@iancharest)

**Comment by 265091z** [comment]
In other words, is it due to the code inside GLMsingle?

**Comment by iancharest** [comment, marked as answer]
Hi,

Yes, in the python version.

I'm sure that it will work if you test the Matlab implementation. We're
working on fixing the python code.

ian

**Reply by 265091z** [reply]
I understand.

Thank you for your response.
